JourneysSquad/JourneysSquad/View/BillRowView.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+/// A view displaying information of the one bill.
+struct BillRowView: View {
+    /// The view model responsible for providing data to the view.
+    @ObservedObject var viewModel: BillRowViewModel
+
+    var body: some View {
+        HStack {
+            VStack(alignment: .leading) {
+                Text(viewModel.dummyDescription)
+                    .lineLimit(1)
+                    .font(.system(size: 14))
+
+                Text(viewModel.formattedParticipants)
+                    .lineLimit(1)
+                    .foregroundColor(.gray)
+                    .font(.system(size: 14))
+            }
+            .padding(Edge.Set.trailing, 120)
+
+            Text(viewModel.dummySumOfBill)
+                .font(.system(size: 14))
+                .lineLimit(1)
+                .frame(maxWidth: 50)
```

Comment:
```suggestion
                .frame(maxWidth: 50)
                .minimumScaleFactor(0.6)
```
I think it could be better to use `minimumScaleFactor` to increase the count of visible digits in big numbers.

Reply:
It seems to me that on small screens the text may turn out to be unreadable with this modifier.

Reply:
@levvaa 
We can set the font `body` for this field and decrease that with `minimumScaleFactor`.

Reply:
I didn't use minimumScaleFactor, because it can shows very little in small screens. Meanwhile, i used body fonts how you recommended.